common/src/main/java/com/graphicsfuzz/common/util/FragmentShaderJobToVertexShaderJob.java

```diff
@@ -77,6 +81,27 @@ public static ShaderJob convertShaderJob(ShaderJob sj) throws Exception {
     TranslationUnit vertexShader = sj.getFragmentShader().get();
     vertexShader.setShaderKind(ShaderKind.VERTEX);
 
+    // Analyse the shader for used features (can't use anonymous class due to member access)
+    class Analysis extends StandardVisitor {
```

Comment:
Rename to e.g. FragmentBuiltinUsageAnalysis.

Also make the class private (I think that should work and if so it is preferred).

Reply:
renamed, but private was not allowed